### PR TITLE
Update to Slim 0.7.1

### PR DESCRIPTION
**Pull request by Punzo**
**New features**
* Support DICOM Advanced Blending Presentation State to parametrize the display of multiplexed IF microscopy images;
* Add key bindings for annotations tools;
* Enable configuration of tile preload;
* Enable configuration of annotation geometry type per finding;
* Expose equipment metadata in user interface.

**Enhancements**
* Improve default presentation of multiplexed IF microscopy images in the absence of presentation state instances;
* Correctly configure DCM4CHEE Archive to use reverse proxy URL prefix for BulkDataURI in served metadata;
* Enlarge display settings interfaces and add input fields for opacity, VOI limits, and colors;
* Update dicom-microscopy-viewer version to use web workers for frame decoding/transformation operations;
* Add button for user logout;
* Disable optical path selection when a presentation state has been selected.

**Bug fixes**
* Fix parsing of URL path upon redirect after successful authentication/authorization;
* Fix configuration of optical path display settings when switching between presentation states;
* Fix caching of presentation states and for selection via drop-down menu.

**Security**
* Update dependencies with critical security issues.

### CODE REVIEW

**Comment by fedorov** [comment]
Per discussions with @hackermd, we will need to update Slim sandbox and test it against the updated HTAN with presentation states. Let's wait merging this PR until we are done with that sandbox testing.

**Comment by hackermd** [comment]
@Punzo thanks for preparing the PR. Could you please remove the configuration files in `.github/workflows`? The deploy pipelines should not be run on the fork

**Comment by hackermd** [comment]
@Punzo I just released version 0.7.1. Could you kindly update the PR to include the recent bug fixes?

**Comment by Punzo** [comment]
> @Punzo I just released version 0.7.1. Could you kindly update the PR to include the recent bug fixes?

done in https://github.com/ImagingDataCommons/slim/pull/61/commits/2152eeb7ad6fbc4c6e98934c767414358d970287

File: src/components/SlideViewer.tsx [hunk 1]
```diff
@@ -2455,6 +2456,7 @@ class SlideViewer extends React.Component<SlideViewerProps, SlideViewerState> {
     )
 
     let presentationStateMenu
+    console.log('DEBUG: ', this.state.presentationStates)
```
Comment (hackermd) [inline src/components/SlideViewer.tsx:2459]: I again forgot a debug message :(

**Comment by wlongabaugh** [comment]
@Punzo @hackermd I have merged to master, but the version now deployed reads 0.7.0, not 0.7.1.

**Comment by Punzo** [comment]
@wlongabaugh @hackermd done in https://github.com/ImagingDataCommons/slim/pull/62

**Comment by fedorov** [comment]
> I have merged to master, but the version now deployed reads 0.7.0, not 0.7.1.

@wlongabaugh as mentioned in https://github.com/ImagingDataCommons/slim/pull/61#issuecomment-1144842858 above, we planned to test this in a sandbox against the re-converted TCGA/CPTAC/NLST and HTAN with presentation states before merging. It's not a huge deal here I guess, but going forward we should try to make sure this kind of communication does not slip through the cracks.

**Pull request merged**